wikimedia/wikipedia-ios
# Native Editor - VoiceOver improvements

**Phabricator:** 
https://phabricator.wikimedia.org/T347870#9500666

### Notes
This PR attempts to smooth over some of the editor VoiceOver handling.

### Test Steps
1. Navigate the editor via voice over.
2. Confirm focusing the text view reads out "Wikitext editor"
3. Double-tap to activate. Depending on how fast your device is, you may hear "Loading editor text" before the text view is activated. This attempts to smooth over the complaint in the Phab task comment.
4. Confirm opening the image wizard via VoiceOver focuses on the Back button first (Before focus seemed stuck on editor text view underneath).
5. Confirm an edit notice focuses on the "Edit notices" title first.
